src/app/Page.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '@/theme/mixins';
+
+.one {
```

Comment:
Maybe such class names like one, two, three etc might be confusing, better to use more descriptive and meaningful names. My suggestion: heading-primary, heading-secondary, subtitle, card-title, accent-text, main-text, big-btn-text. medium-btn-text, small-text or your variants.

Reply:
There were examples for teammates of using mixins, we will get rid of them very soon, renamed anyway